### PR TITLE
Remove 'experimental' from high-availability, and '.' from etcd bullet

### PR DESCRIPTION
**Pull request by natereid72**
s/controller-managing/controller-manager
s/Apiserver/API server
s/Assuming/Assumes

### CODE REVIEW

File: content/en/docs/tasks/debug-application-cluster/debug-cluster.md [hunk 1]
```diff
@@ -100,11 +100,11 @@ This is an incomplete list of things that could go wrong, and how to adjust your
 - Action: Use IaaS providers reliable storage (e.g. GCE PD or AWS EBS volume) for VMs with apiserver+etcd
   - Mitigates: Apiserver backing storage lost
 
-- Action: Use (experimental) [high-availability](/docs/admin/high-availability) configuration
+- Action: Use [high-availability](/docs/admin/high-availability) configuration
   - Mitigates: Master VM shutdown or master components (scheduler, API server, controller-managing) crashing
```
Comment (justaugustus) [inline content/en/docs/tasks/debug-application-cluster/debug-cluster.md]:
```suggestion
  - Mitigates: Control plane node shutdown or control plane components (scheduler, API server, controller-manager) crashing
```

File: content/en/docs/tasks/debug-application-cluster/debug-cluster.md [hunk 2]
```diff
@@ -100,11 +100,11 @@ This is an incomplete list of things that could go wrong, and how to adjust your
...
     - Will tolerate one or more simultaneous node or component failures
   - Mitigates: Apiserver backing storage (i.e., etcd's data directory) lost
-    - Assuming you used clustered etcd.
+    - Assuming you used clustered etcd
```
Comment (justaugustus) [inline content/en/docs/tasks/debug-application-cluster/debug-cluster.md]:
```suggestion
    - Assumes HA (highly-available) etcd configuration
```

File: content/en/docs/tasks/debug-application-cluster/debug-cluster.md [hunk 3]
```diff
@@ -100,11 +100,11 @@ This is an incomplete list of things that could go wrong, and how to adjust your
 - Action: Use IaaS providers reliable storage (e.g. GCE PD or AWS EBS volume) for VMs with apiserver+etcd
   - Mitigates: Apiserver backing storage lost
 
-- Action: Use (experimental) [high-availability](/docs/admin/high-availability) configuration
+- Action: Use [high-availability](/docs/admin/high-availability) configuration
   - Mitigates: Master VM shutdown or master components (scheduler, API server, controller-managing) crashing
     - Will tolerate one or more simultaneous node or component failures
   - Mitigates: Apiserver backing storage (i.e., etcd's data directory) lost
```
Comment (justaugustus) [inline content/en/docs/tasks/debug-application-cluster/debug-cluster.md]:
```suggestion
  - Mitigates: API server backing storage (i.e., etcd's data directory) lost
```

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-17254--kubernetes-io-master-staging.netlify.com/docs/tasks/debug-application-cluster/debug-cluster/

File: content/en/docs/tasks/debug-application-cluster/debug-cluster.md (hunk 1 above)
Comment (kbhawkey) [reply content/en/docs/tasks/debug-application-cluster/debug-cluster.md]: Hello @natereid72. The changes seem okay. Would it be possible for you to incorporate the comments from @justaugustus? Thanks!

File: content/en/docs/tasks/debug-application-cluster/debug-cluster.md (hunk 3 above)
Comment (kbhawkey) [reply content/en/docs/tasks/debug-application-cluster/debug-cluster.md]: @natereid72 , If possible replace all instances of `Apiserver` with `API server`.

**Comment by bradtopol** [comment]
Hi @justaugustus can you serve as the technical reviewer for this? Just add a /lgtm when you are happy with it and I'll take care of the rest.  
/cc @justaugustus

**Pull request merged**